src/plugins/unified_search/public/filter_bar/filter_editor/phrase_value_input.tsx

```diff
@@ -6,44 +6,34 @@
  * Side Public License, v 1.
  */
 
-import { EuiFormRow } from '@elastic/eui';
 import { InjectedIntl, injectI18n } from '@kbn/i18n-react';
 import { uniq } from 'lodash';
 import React from 'react';
 import { withKibana } from '@kbn/kibana-react-plugin/public';
 import { GenericComboBox, GenericComboBoxProps } from './generic_combo_box';
 import { PhraseSuggestorUI, PhraseSuggestorProps } from './phrase_suggestor';
 import { ValueInputType } from './value_input_type';
-import { getFieldValidityAndErrorMessage } from '../../utils/helpers';
 
-interface Props extends PhraseSuggestorProps {
+interface PhraseValueInputProps extends PhraseSuggestorProps {
   value?: string;
   onChange: (value: string | number | boolean) => void;
   intl: InjectedIntl;
   fullWidth?: boolean;
+  compressed?: boolean;
+  disabled?: boolean;
+  isInvalid?: boolean;
 }
 
-class PhraseValueInputUI extends PhraseSuggestorUI<Props> {
+class PhraseValueInputUI extends PhraseSuggestorUI<PhraseValueInputProps> {
   public render() {
-    const { isInvalid, errorMessage } = getFieldValidityAndErrorMessage(
```

Comment:
@nlatipov does this change means that new `filter_builder` component works without validation?

Reply:
Thanks Aliaksei for bringing this to my attention. Of course the validation will be added to the filter builder